app/models/shelter.rb

```diff
@@ -8,15 +8,6 @@ def approved_pet?
     pets.where(adopted?: "Pending").empty? == false
   end
 
-  def self.empty_params(shelter_param)
```

Comment:
replacing this check of the params with the `error.full_message.to_sentence` call on the object allows us reduce the amount of ruby code to its minimum in the model